src/Thujohn/Twitter/Traits/UserTrait.php

```diff
@@ -7,7 +7,7 @@
 trait UserTrait
 {
     /**
-     * Returns fully-hydrated user objects for up to 100 users per request, as specified by comma-separated values passed to the user_id and/or screen_name parameters.
+     * Returns fully-hydrated user objects for up to 10 users per request, as specified by comma-separated values passed to the user_id and/or screen_name parameters.
```

Comment:
Can you fix this comment, and also method why it uses post now :)

Reply:
Sure)